docs/user_guides/single_table/copulagan.rst

```diff
@@ -756,14 +760,18 @@ dataset are within 0 and 1, ``CopulaGAN`` will not be able to set this value to
 
 .. note::
 
-    Currently, conditional sampling works through a rejection sampling process,
-    where rows are sampled repeatedly until one that satisfies the conditions is
-    found. In case you are running into a ``Could not get enough valid rows within
-    x trials`` or simply wish to optimize the results, there are three parameters
-    that can be fine-tuned: ``max_rows_multiplier``, ``max_retries`` and ``float_rtol``.
-    More information about these parameters can be found in the `API section
-    <https://sdv.dev/SDV/api_reference/tabular/api/sdv.tabular.copulagan.CopulaGAN.sample.
-    html>`__.
+    Conditional sampling works through a rejection sampling process, where
+    rows are sampled repeatedly until one that satisfies the conditions is found.
+    In case you are not able to sample enough valid rows, update the related parameters:
+    increasing ``max_tries`` or increasing ``batch_size_per_try``.
+    More information about these paramters can be found in the `API section
+    <https://sdv.dev/SDV/api_reference/tabular/api/sdv.tabular.copulagan.CopulaGAN.
+    sample_conditions.html>`__
+
+    If you have many conditions that cannot easily be satisified, consider switching
+    to the `GaussianCopula model <https://sdv.dev/SDV/api_reference/tabular/api/sdv.
```

Comment:
I suggest we link to the User Guide for GaussianCopula instead of the API reference.